framework/test/unit/specs/modules/keys/register_asset.spec.ts

```diff
@@ -0,0 +1,340 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import { randomBytes } from 'crypto';
+import { codec } from '@liskhq/lisk-codec';
+import { hash } from '@liskhq/lisk-cryptography';
+import { Transaction, transactionSchema } from '@liskhq/lisk-chain';
+import { RegisterAsset } from '../../../../../src/modules/keys/register_asset';
+import { createFakeDefaultAccount, StateStoreMock } from '../../../../utils/node';
+import * as fixtures from './fixtures.json';
+
+let decodedMultiSignature: any;
```

Comment:
variable definition should be in the `describe`

Reply:
Done.